[CHORE]: PyPI package verify failure

### 🔧 Chore Summary

There's a mismatch on the version control (git) and the PyPI package. This is happening as the JS bundle is being built at at package build time and included in the wheelfile, but it is not (neither should it be) on the git repo. This causes the following `make verify` issue:

```
lists of files in version control and sdist do not match!
missing from VCS:
  mcpgateway/static/bundle-Djs52C0w.js
make: *** [verify] Error 1
```

---

### 🧱 Area Affected

Choose the general area(s) that this chore affects:

- [ ] GitHub Actions / CI Pipelines
- [ ] Pre-commit hooks / linters
- [ ] Formatting (black, isort, ruff, etc.)
- [ ] Type-checking (mypy, pyright, pytype, etc.)
- [ ] Dependency cleanup or updates
- [ ] Build system or `Makefile`
- [ ] Containerization (Docker/Podman)
- [ ] Docs or spellcheck
- [ ] SBOM, CVE scans, licenses, or security checks
- [X] Other: PyPI package

---

### ⚙️ Context / Rationale

We need to get our dist package verification clean and correct.

---

### 📦 Related Make Targets

Reference any relevant Makefile targets that are involved, if applicable. Ex:

- `make dist` - Builds the PyPI package
- `make verify` - Verify the package integrity and correctness

---

### 📋 Acceptance Criteria

Define what "done" looks like for this task.

- [ ] CI passes with no regressions
- [ ] The PyPI package runs with no errors
- [ ] `make verify` completes all the verifications successfully

---

### 🧩 Additional Context

(Optional) Include any configs, environment quirks, or dependencies (e.g. Python, Node, Docker, CI secrets).

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -373,6 +373,7 @@ ignore = [
     ".github/**",
     "Makefile",
     "mcpgateway/admin_ui/**",
+    "mcpgateway/static/bundle-*.js",
 ]
 
 [tool.black]

